src/Microsoft.HttpRepl.IntegrationTests/Mocks/ConsoleManagerStub.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using Microsoft.Repl.ConsoleHandling;
+using Moq;
+
+namespace Microsoft.HttpRepl.IntegrationTests.Mocks
+{
+    public class ConsoleManagerStub : IConsoleManager
+    {
+        public Point Caret => default;
+
+        public Point CommandStart => default;
+
+        public int CaretPosition => default;
+
+        public IWritable Error => default;
+
+        public bool IsKeyAvailable => default;
+
+        public bool AllowOutputRedirection => default;
+
+        public bool IsCaretVisible { get => default; set { } }
+
+        public IDisposable AddBreakHandler(Action onBreak) => default;
+
+
```

Comment:
nit: empty line

Reply:
Fixed in https://github.com/aspnet/HttpRepl/pull/74/commits/af55ecc8092914f16ced3d5fd83db4b96b4de1de